Apps/WebClient/src/WebClient.csproj

```diff
@@ -100,7 +100,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="wwwroot\dist\**" />
-      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
+      <ResolvedFileToPublish Include="@(DistFiles-&gt;'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```

Comment:
this looks odd. Is it supposed to be encoded?

Reply:
Visual studio automatically does that, We need to check if works for you and Brad!